### DISCUSSION TITLE
Show instance selection box on mouse over

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#1879] [Ideas]
Instance selection at the moment is a bit sketchy since we don't capture clicks within the bounding box of the instance, so you only see the selection box around it when clicking on a node. This means that the mechanism for instance selection can be a bit unintuitive for new users to discover.

Knowing how to do this is important since deleting instances requires instance selection, and comes up early in the workflow. The other case where we're using instance selection is for assigning tracks -- this comes up later in the workflow, by which point users will have discovered how to select instances. Yet another case that will come up soon will be selecting instances to assign them to instance groups as part of the upcoming multiview annotation functionality.

A great visual indicator would be to show the selection box (perhaps with thinner edges and/or with some transparency) when mousing over the instance.

**Comment by talmo** [comment]
Implemented in #2055.

**Comment by roomrys** [comment]
Long awaited, but finally integrated. This issue has been fixed in the SLEAP 1.4.1 release!
